<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>草莓味的SR620记录仪</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>banana.jpg</normaloff>banana.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyComboBox" name="SerialSelectBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>选择串口…</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TimeBaseSelectBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="MatplotlibWidget" name="MatplotlibWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SerialPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>292</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>已停止</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header location="global">MatplotlibWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">myComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
